pkg/kubelet/images/puller.go

```diff
@@ -40,14 +40,22 @@ var _, _ imagePuller = &parallelImagePuller{}, &serialImagePuller{}
 
 type parallelImagePuller struct {
 	imageService kubecontainer.ImageService
+	tokens       chan bool
```

Comment:
If you don't need a value, you can consume less space with a `struct{}` than a `bool`.

```suggestion
	tokens       chan struct{}
```

Reply:
good suggestion!